dts/arm/nxp/nxp_k6x.dtsi

```diff
@@ -65,6 +65,34 @@
 
 			flash0: flash@0 {
 				reg = <0 0x100000>;
+
+				partitions {
```

Comment:
How about adding a comment about zephyr,code-partition here.  Since if I was to look at the dts I wouldn't assume that code used the full flash w/o zephyr,code-partition.